lib/apartment/adapters/sqlite3_adapter.rb

```diff
@@ -13,6 +13,9 @@ def initialize(config)
         @default_dir = File.expand_path(File.dirname(config[:database]))
 
         super
+
+        Apartment.connection_handler = ActiveRecord::ConnectionAdapters::ConnectionHandler.new
+        reset
```

Comment:
OK, was just suggesting since my yet to come pull request let people change configuration which is likely to happen only for connection, not schemas.